crutest/src/main.rs

```diff
@@ -297,6 +279,25 @@ fn history_file<P: AsRef<Path>>(file: P) -> PathBuf {
     out
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, clap::Args)]
+struct RandReadWriteWorkload {
+    /// Size in blocks of each IO
+    #[clap(long, default_value_t = 1, action)]
+    io_size: usize,
+    /// Number of outstanding IOs at the same time.
+    #[clap(long, default_value_t = 1, action)]
+    io_depth: usize,
+    /// Number of seconds to run
+    #[clap(long, default_value_t = 60, action)]
+    time: u64,
+    /// Completely fill the region with random data first
+    #[clap(long)]
+    fill: bool,
+    /// Print the guest log to `stderr`
```

Comment:
This changes the default logging for the upstairs to log only errors?  Or does
it still print all messages but only to stderr?  It suggests the latter, but the code
below makes me think we are only printing error level logs?